packages/views/__tests__/errorBoundary.test.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * @jest-environment jsdom
+ */
+import * as React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import { bound } from "@frui.ts/helpers";
+import ErrorBoundary, { ErrorBoundaryProps } from "../src/errorboundary";
+
+function Boo({ title }: { title: string }): JSX.Element {
+  throw new Error(title);
+}
+
+function Bam(): JSX.Element {
+  return <Boo title={"test"} />;
+}
+
+class TestApp extends React.Component<ErrorBoundaryProps, { error: boolean }> {
```

Comment:
What does this class do? Is it useful?

Reply:
It tests normal render and than via button simulate error inside view layer and errorBoundary component reaction on it.

Reply:
Good idea. Could you please update the code so that the purpose is more obvious? E.g., add a label to the Error button like "Raise error".